egs2/TEMPLATE/svs1/README.md

```diff
@@ -281,35 +281,36 @@ $  --pretrained_model /exp/xiaoice-2-24-250k/500epoch.pth:svs:svs.fftsinger \
 
 
 ### VISinger training
-The VITS config is hard coded for 22.05 khz or 44.1 khz and use different feature extraction method. (Note that you can use any feature extraction method but the default method is linear_spectrogram.) If you want to use it with 24 khz or 16 khz dataset, please be careful about these point.
+The VISinger config is hard coded for 44.1 khz and use different feature extraction method. (Note that you can use any feature extraction method but the default method is `fbank`.) If you want to use it with 24 khz or 16 khz dataset, please be careful about these point.
```

Comment:
Can we relax the sampling rate requirement? I think 44.1khz is still a very high standard to be used in many applications.

Reply:
> 

How about adding 22.05 khz config files in both opencpop and ofuton? How should I name them?

Reply:
I think we can just add a 22.05 suffix to those. Just in case someone would be interested in that

Reply:
For pre-trained models, we can just do the 44.1 if the dataset supports (future action items)